e2e_tests/plugins/index.js

```diff
@@ -1,17 +1,9 @@
-// ***********************************************************
-// This example plugins/index.js can be used to load plugins
-//
-// You can change the location of this file or turn off loading
-// the plugins file with the 'pluginsFile' configuration option.
-//
-// You can read more here:
-// https://on.cypress.io/plugins-guide
-// ***********************************************************
+const webpack = require('@cypress/webpack-preprocessor')
 
-// This function is called when a project is opened or re-opened (e.g. due to
-// the project's config changing)
-
-module.exports = (on, config) => {
-  // `on` is used to hook into various events Cypress emits
-  // `config` is the resolved Cypress config
+module.exports = on => {
```

Comment:
To be able to import our redux actions we need to use webpack (handles typescript/aliased imports/etc)

Reply:
This comment is useful when reading code, I think it should be added in source